### ISSUE TITLE
Impossible to Select clipboard to paste

### ISSUE THREAD

**Issue by moocan** [stef-levesque/vscode-multiclip#34]
Hello,

Installed Multiclip on VisualStudio Code 
```
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:48:44.518Z
```

There is no `Ctrl+Alt+v` and Command Palette does no show any way to select a clipboard (screen capture attached)

![multiclip](https://user-images.githubusercontent.com/21237706/109359318-94f50080-7885-11eb-8473-0e7a509638f4.png)

If I list Multiclip Buffer I can see items in the buffer, past the last one copied but can't select any previous items.

Regards

**Comment by jasonbrancazio** [comment]
I have the same issue. As a workaround, create a custom keyboard shortcut for Multiclip List Buffer, and use the arrow keys to select the item to paste.

**Comment by ferm10n** [comment]
Yeah for some reason on a fresh install the keybinding for picking a clipboard (Ctrl + alt + v) is not set

**Comment by tjx666** [comment]
try https://github.com/tjx666/vscode-clipboard-master

**Comment by hellmrf** [comment]
Just use [`clipboard-manager`](https://marketplace.visualstudio.com/items?itemName=EdgardMessias.clipboard-manager).

---

_(Original answer:)_

The default keybindings aren't particularly great. I've changed -- and recommend changing -- the `Ctrl+Shift+V` keybinding to the `multiclip.list` command, so we have:

 - `Ctrl+V`: pastes the most recent item, which is the usual, default behavior (`multiclip.regularPaste`);
 - `Ctrl+Shift+V`: opens the clipboard history so we can choose what to paste (`multiclip.list`).

The [Keyboard Shortcuts](https://code.visualstudio.com/docs/configure/keybindings) settings page should look something like this:

![Image](https://github.com/user-attachments/assets/a96f6cd3-7a18-46f5-9052-c449123cffaf)

Instead of reassigning them through the UI, you can paste the following into your [`keybindings.json`](https://code.visualstudio.com/docs/configure/keybindings#_advanced-customization):

```jsonc
[
    {  // ...
       // Other keybindings...
    },   // (Don't forget this comma!)
    {
        "key": "ctrl+shift+v",
        "command": "-multiclip.paste",
        "when": "editorTextFocus"
    },
    {
        "key": "ctrl+shift+v",
        "command": "multiclip.list",
        "when": "editorTextFocus"
    }
]
```
